chain/stmgr/stmgr.go

```diff
@@ -1120,3 +1123,11 @@ func (sm *StateManager) GetCirculatingSupply(ctx context.Context, height abi.Cha
 
 	return csi.FilCirculating, nil
 }
+
+func (sm *StateManager) GetNtwkVersion(ctx context.Context, height abi.ChainEpoch) runtime.NetworkVersion {
+	if build.UpgradeBreezeHeight == 0 || height <= build.UpgradeBreezeHeight {
```

Comment:
I think 0 should mean, use the Version1, unless everything breaks.

Reply:
Done, everything shouldn't break (the actual changes are trivial)